### PR TITLE
Add nix-output-monitor

### PR DESCRIPTION
**Pull request by j4m3s-s**
Add link to another nix tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,7 @@
 * [nix-init](https://github.com/nix-community/nix-init) - Generate Nix packages from URLs with hash prefetching, dependency inference, license detection, and more.
 * [nix-melt](https://github.com/nix-community/nix-melt) - A ranger-like flake.lock viewer.
 * [nix-prefetch](https://github.com/msteen/nix-prefetch) - A universal tool for updating source checksums.
+* [nix-output-monitor](https://git.maralorn.de/nix-output-monitor/) - A tool to get additional information while building.
```
Comment (cyntheticfox) [inline README.md:113]:
```suggestion
* [nix-output-monitor](https://git.maralorn.de/nix-output-monitor/) - A tool to produce useful graphs and statistics when building derivations.
```

The original description is correct, both factually and grammatically, but kinda undersells the tool, so I'd recommend the above word choice, but that's up to you.

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
Word choice recommendation is optional, but please move the entry up one line to before `nix-prefetch` to maintain alphabetical ordering

**Comment by j4m3s-s** [comment]
And here I thought I did it alphabetically ...
Fixed it. I used your description which is more descriptive of the tool, thanks for the suggestion.

**Review by cyntheticfox — APPROVED** [review]
Looks good now

**Pull request merged**